src/components/Filters.js

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import {sortirajAbecedno} from '../shared/helpers'
 import './Filters.css'
 
 const Filters = ({ authorImages, authors, language, setPhrase, setAuthor }) => {
-  const sortirano = [...authors].sort(sortirajAbecedno)
+  const sortirano = [...authors].sort()
```

Comment:
što si uklonio sortirajAbecedno, šta se dešava bez nje?

Reply:
mislim da ta funkcija nicemu nije sluzila to jest radila je isto sto i sort radi po difoltu

Reply:
sort sortira niz prostih vrednosti, a ova funkcija sortira niz objekata

Reply:
izvini, ipak si bio u pravu, ta funkcija nicemu nije sluzila, verovatno je ostala. autori su samo stringovi, nisu objekti. izvini još jednom, moja greška.